modules/fts/pages/fts-accessing-managing-full-text-indexes.adoc

```diff
@@ -0,0 +1,23 @@
+= Accessing and Managing Full Text Indexes
+
+[abstract]
+Full Text indexes, which are available under the *Search* tab of the Couchbase Web Console, are another form of the index provided by the Search service.
+
+Full Text indexes are special-purpose indexes that contain targets derived from the textual contents of the documents within one or more buckets or collections from the buckets.
+For more information about different types of indexes, see xref:learn:services-and-indexes/indexes/indexes.adoc[Indexes].
+
+Full Text Indexes are accessed from the *Search* tab: left-click on this to display the *Full Text Search* panel, which contains a tabular presentation of currently existing indexes, with a row for each index.
+(See xref:fts-searching-from-the-ui.adoc[Searching from the UI] for a full illustration.) To manage an index, left-click on its row.
+The row expands, as follows:
+
+[#fts_index_management_ui]
+image::fts-index-management-ui.png[,820,align=left]
+
+Three buttons are displayed:
+
+* [.ui]*Delete* causes the current index to be deleted.
+* [.ui]*Clone* brings up the *Clone Index* screen, which allows a copy of the current index to be modified as appropriate and saved under a new name.
+* [.ui]*Edit* brings up the *Edit Index* screen, which allows the index to be modified.
+Saving modifications causes the index to be rebuilt.
```

Comment:
Not necessarily ..

Modifying the index definition **_CAN_** cause indexes to be rebuilt.

There's also another option called "Quick Edit" that goes to the quick editor for an index definition.